drag_and_drop_v2/public/js/translations/ar/text.js

```diff
@@ -1,8 +1,4 @@
 
-            (function(global){
-                var DragAndDropI18N = {
```

Comment:
any specific reason for removing this namespace?

Reply:
Added again. Thanks for highlighting.